dashboard/src/components/InstanceListView/index.tsx

```diff
@@ -75,39 +76,61 @@ export class InstanceListView extends React.PureComponent<InstanceListViewProps>
     const { error, isInstalled, brokers, instances, classes } = this.props;
 
     return (
-      <div className="InstanceList">
-        <h1 className="margin-b-reset">Service Instances</h1>
-        <hr />
-        {isInstalled ? (
-          <div>
-            {error ? (
-              this.renderError()
-            ) : brokers.length > 0 ? (
-              <div>
-                <div className="row align-center">
-                  <div className="col-8">
-                    <p>Service instances from your brokers:</p>
-                  </div>
-                  <div className="col-4 text-r">
-                    <Link to={`/services/classes`}>
-                      <button className="button button-accent">Provision New Service</button>
-                    </Link>
-                  </div>
-                </div>
-                {instances.length > 0 ? (
-                  <InstanceCardList instances={instances} classes={classes} />
-                ) : (
-                  <div>No service instances are found.</div>
-                )}
+      <section className="InstanceList">
+        <header className="InstanceList__header">
+          <div className="row padding-t-big collapse-b-phone-land">
+            <div className="col-8">
+              <h1 className="margin-v-reset">Service Instances</h1>
+            </div>
+            {instances.length > 0 && (
+              <div className="col-4 text-r align-center">
+                <Link to={`/services/classes`}>
+                  <button className="button button-accent">Deploy Service Instance</button>
+                </Link>
               </div>
-            ) : (
-              <ServiceBrokersNotFoundAlert />
             )}
           </div>
-        ) : (
-          <ServiceCatalogNotInstalledAlert />
-        )}
-      </div>
+          <hr />
+        </header>
+        <main>
+          {isInstalled ? (
+            <div>
+              {error ? (
+                this.renderError()
+              ) : brokers.length > 0 ? (
+                <div>
+                  {instances.length > 0 ? (
+                    <InstanceCardList instances={instances} classes={classes} />
+                  ) : (
+                    <MessageAlert
+                      header={"Provision External Services from the Kubernetes Service Catalog"}
+                      children={
```

Comment:
I might have missed this in your other PR, but `children` is a special prop in React and should be passed as child HTML elements. For example:
```
<MessageAlert header="Provision External Services from the Kubernetes Service Catalog">
  <div>...</div>
</MessageAlert>
```

Can you please go back and update this in AppList and FunctionList?